packages/cactus-plugin-keychain-azure-kv/src/main/typescript/plugin-keychain-azure-kv-remote-adapter.ts

```diff
@@ -0,0 +1,156 @@
+import type { Server } from "http";
+import type { Server as SecureServer } from "https";
+
+import type { Express } from "express";
```

Comment:
@jagpreetsinghsasan Thank you for adding `type` keyword, that's awesome. What this does is that you you can move the `express` dependency to `devDependencies` in the package.json file as well (and then run the configure script again to update the lock file accordingly)